apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Shuffler.java

```diff
@@ -20,331 +20,24 @@
-import com.apollocurrency.aplwallet.apl.core.service.state.impl.ShufflingServiceImpl;
-import com.apollocurrency.aplwallet.apl.core.shard.DbHotSwapConfig;
-import com.apollocurrency.aplwallet.apl.core.transaction.FeeCalculator;
-import com.apollocurrency.aplwallet.apl.core.transaction.messages.Attachment;
-import com.apollocurrency.aplwallet.apl.core.transaction.messages.ShufflingAttachment;
-import com.apollocurrency.aplwallet.apl.core.transaction.messages.ShufflingCancellationAttachment;
-import com.apollocurrency.aplwallet.apl.core.transaction.messages.ShufflingRegistration;
-import com.apollocurrency.aplwallet.apl.core.transaction.messages.ShufflingVerificationAttachment;
-import com.apollocurrency.aplwallet.apl.crypto.Convert;
 import com.apollocurrency.aplwallet.apl.crypto.Crypto;
-import com.apollocurrency.aplwallet.apl.util.Constants;
-import com.apollocurrency.aplwallet.apl.util.injectable.PropertiesHolder;
-import lombok.extern.slf4j.Slf4j;
-import org.slf4j.Logger;
-
-import javax.enterprise.event.Observes;
-import javax.enterprise.inject.spi.CDI;
-import javax.inject.Singleton;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-
-import static org.slf4j.LoggerFactory.getLogger;
 
 public final class Shuffler {
-    private static final Logger LOG = getLogger(Shuffler.class);
-
-    // TODO: YL remove static instance later
-    private static final Map<String, Map<Long, Shuffler>> shufflingsMap = new HashMap<>();
-    private static final Map<Integer, Set<String>> expirations = new HashMap<>();
-    private static PropertiesHolder propertiesLoader = CDI.current().select(PropertiesHolder.class).get();
-    private static final int MAX_SHUFFLERS = propertiesLoader.getIntProperty("apl.maxNumberOfShufflers");
-    private static TransactionProcessor transactionProcessor = CDI.current().select(TransactionProcessorImpl.class).get();
-    private static Blockchain blockchain = CDI.current().select(BlockchainImpl.class).get();
-    private static GlobalSync globalSync = CDI.current().select(GlobalSync.class).get();
-    private static ShufflingService shufflingService = CDI.current().select(ShufflingServiceImpl.class).get();
-    private static FeeCalculator feeCalculator = new FeeCalculator();
-    private static BlockchainProcessor blockchainProcessor;
-    private static AccountService accountService;
     private final long accountId;
     private final byte[] secretBytes;
     private final byte[] recipientPublicKey;
     private final byte[] shufflingFullHash;
     private volatile Transaction failedTransaction;
     private volatile AplException.NotCurrentlyValidException failureCause;
 
-    private Shuffler(byte[] secretBytes, byte[] recipientPublicKey, byte[] shufflingFullHash) {
+    public Shuffler(byte[] secretBytes, byte[] recipientPublicKey, byte[] shufflingFullHash) {
```

Comment:
Pls move it into correct package:
`com.apollocurrency.aplwallet.apl.core.entity.state.shuffling`